connection.js

```diff
@@ -589,9 +578,8 @@ class Connection {
     handlePong(data) {{{
         let now = Date.now();
         let latency = now - data.client;
-        let drift = ((now-latency/2) - data.server);
         this.network_latency = latency;
-        this.clock_drift = drift;
+        this.clock_drift = ((now-latency/2) - data.server);
```

Comment:
`this.clock_drift = (now-latency/2) - data.server;` would work too.

Reply:
:+1: